### ISSUE TITLE
Firewall vor Space

### ISSUE THREAD

**Issue by geregeltkrieger** [mittwald/spaces-feedback#11]
Ich würde dem Space erlauben können, aber der Space selbst ins Internet darf; im Normalfall muss der ja nur Antworten.
Eine Ausnahme kann die Abfrage von dritten APIs sein, d.h. ich hätte gerne Domains, die ich Whitelisten kann (Bsp: weather,yahoo.com)

**Comment by gerrithartwig** [comment]
Hey Alexander,

danke für Dein Feedback. Nehme ich gerne als Featurewunsch mit auf.

Eine Frage hätte ich aber noch... geht es Dir darum, dass nur bestimmte Services auf deinen Space zugreifen können oder soll Deine Applikation nur auf bestimmte Services zugreifen können?

Viele Grüße
Gerrit Hartwig

**Comment by geregeltkrieger** [comment]
Es geht darum, dass ein TYPO3 Projekt typischerweise nicht selbst auf das Internet zugreift; wenn es das doch tut, ist das oft ein Anzeichen von einem Hack. Das will ich damit verhindern, bzw. einschränken.
Keine Regel ohne Ausnahme: es gibt Projekt von uns, die z.B. alle 15 Minuten Börsenkurs von Yahoo lesen- diese müssten dann per Whitelist trotzdem erreichbar sein. Wichtig: die Whitelist muss Name-Bades sein (IPs von Diensten ändern sich schon ab und an Mal).

**Comment by gerrithartwig** [comment]
Alles klar. Danke für die Erläuterung.

Wir schauen, was wir da machen können. Was mir spontan einfällt, was etwas tricky werden kann ist, dass ja auch je nach Anwendung bspw. die Extenesion Manager o.ä. auch nach extern telefonieren wollen.

Grüße
Gerrit

**Comment by geregeltkrieger** [comment]
Richtig- aber genau das will ich ja verhindern, bzw.: genauer: kontrollieren. Gerade bei der Übernahme eine fremden Extension kann ich ja erst Mal nicht sehen, was die so alles Hause schickt. Daher müsste die Firewall einfach (=per API) an und abzuschalten sein.
